src/jinacld_tools/aws/services/cloudwatch.py

```diff
@@ -43,6 +49,8 @@ def fetch(self, ec2ARN):
         ec2 = boto3.resource('ec2', region_name=region,
                 aws_access_key_id=ACCESS_KEY_ID, aws_secret_access_key=SECRET_ACCESS_KEY)
 
+        instance = ec2.Instance(instanceID)
```

Comment:
I would recommend having a new service called `ec2` in a different file from which you can access the `ec2.Instance` object

Reply:
I will create a minimal ec2 service but I have not yet see another way to get the launch_time attribute without calling boto3 resource